config/routes.rb

```diff
@@ -5,4 +5,5 @@
   get  "/about", to: "static_pages#about"
   get  "/contact", to: "static_pages#contact"
   get  "/signup", to: "users#new"
+  resources :users
```

Comment:
Specify the actions for this resource
`resources :users, only: %i(new create show)`